api/validators.py

```diff
@@ -0,0 +1,4 @@
+from django.core.validators import RegexValidator
+
+
+phoneregex = "[0-9]{9}"
```

Comment:
You are not using the `RegexValidator` anywhere :confused: 
You created an string, not a regular expression.

Reply:
Yep, I was doing a couple of things on the same time.

Thanks!